docroot/themes/humsci/humsci_basic/src/scss/objects/_layouts.three_column_w_image.scss

```diff
@@ -51,45 +51,32 @@
   &__main-top {
     display: flex;
     flex-wrap: wrap;
-    align-items: flex-start;
+    justify-content: space-between;
 
-    .hb-three-column-w-image--one-sidebar & {
-      @include grid-media-min('lg') {
-        flex-wrap: nowrap;
-      }
-    }
-
-    .hb-three-column-w-image--no-sidebar & {
-      @include grid-media-min('md') {
-        flex-wrap: nowrap;
-      }
+    @include grid-media-min('md') {
+      flex-wrap: nowrap;
     }
   }
 
   &__main-top-text {
-    width: 100%;
-    margin-bottom: hb-calculate-rems(16px);
+    margin: 0 hb-calculate-rems(16px) hb-calculate-rems(16px) 0;
 
     .hb-three-column-w-image--one-sidebar & {
       @include grid-media-min('lg') {
-        width: 60%;
         display: flex;
         flex-direction: column;
       }
     }
 
     .hb-three-column-w-image--no-sidebar & {
       @include grid-media-min('md') {
-        width: 60%;
         display: flex;
         flex-direction: column;
       }
     }
   }
 
   &__main-top-image {
-    width: 100%;
```

Comment:
hmm I think at mobile, this is ok?

Reply:
This property doesn't do anything now that we are using the Drupal image styles.

Reply:
A note on the functionality of images now that we are using the Drupal image style functionality to select different image styles (sizes and dimensions): Only images medium or larger scale to 100% on mobile. Once the image is larger than the mobile screen size, they scale down to fit. Small images are only 200px width therefore will not scale at mobile. Adding styles to make small image style will cause the image to pixelate which is not recommended.

Image will also wrap and scale at different breakpoints depending upon the length of the text in the title.

Reply:
I will add this to the code base.